artmaker/src/main/java/io/artmaker/composables/StrokePreview.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2024 ArtMaker
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.artmaker.composables
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Path
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.res.dimensionResource
+import io.fbada006.artmaker.R
+import kotlin.math.sin
+
+@Composable
+internal fun StrokePreview(state: StrokeState, modifier: Modifier = Modifier) {
+    Box(
+        modifier = modifier
+            .background(color = Color(248, 246, 240), shape = RoundedCornerShape(dimensionResource(R.dimen.Padding12))),
```

Comment:
For this background color, why do you specify the RGB float values? 
Also put the shape on separate line, I am not sure why Spotless didn't catch this.